code401Challenges/src/main/java/code401Challenges/hashtable/Hashtable.java

```diff
@@ -1,10 +1,52 @@
 package code401Challenges.hashtable;
 
+import java.util.LinkedList;
+
 public class Hashtable {
+    LinkedList<Entry>[] hashTableArray = new LinkedList[100]; // Array of linked lists
+
     // add: takes in both the key and value. This method should hash the key, and add the key and value pair
     // to the table, handling collisions as needed.
+    public boolean add (String key, String value) {
+        // takes a key and turns it into a hashed thing
+        int arrIndex = hash(key);
+        Entry entry = new Entry(key, value);
+
```

Comment:
Here we can instantiate the LinkedList at that position of the array, if it is still null.
```if (hashTableArray[arrIndex] == null) {
            hashTableArray[arrIndex] = new LinkedList<>();
   }```